model-engine/model_engine_server/core/celery/app.py

```diff
@@ -475,6 +475,8 @@ def _get_broker_endpoint_and_transport_options(
         # backoff_policy, etc., then we can expose broker_transport_options in the top-level celery() wrapper function.
         # Going to try this with defaults first.
         out_broker_transport_options["region"] = os.environ.get("AWS_REGION", "us-west-2")
+        out_broker_transport_options["wait_time_seconds"] = 0
```

Comment:
just curious: can you explain how this works?

Reply:
+1, would be good to explain what's going on with these changes, and/or any testing results showing the impact of the changes.